_posts/2020-08-05-hive-acid-original-files.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title:  "Read support for original files of Hive transactional tables in Presto"
+author: Harmandeep Singh, Qubole
+excerpt_separator: <!--more-->
+---
+
+In Presto 331, read support for Hive transactional tables was introduced. It works well if a user creates a new Hive transactional table and reads it from Presto. However,  if an
+existing table is converted to a Hive transactional table, Presto would fail to read data from such a table because  read support for original files was missing. Original files are
+those files in a Hive transactional table that existed before the table was converted into a Hive transactional table and until version 340, Presto expected all files in a Hive
```

Comment:
340 -> ...

(not merged yet, sorry for the delay)

this applies in other places too

Reply:
okay, I will modify the version when finalised.

Reply:
341 has all the changes, am I right?

Reply:
341 has all the changes already merged, am I right?

Reply:
I can see the commit mentioned in 340 release notes, sorry I am confused, shouldn't it be 340 then?

Reply:
You're right. I mixed this up with a different change.